TestFoundation/TestNSFileManager.swift

```diff
@@ -114,6 +115,32 @@ class TestNSFileManager : XCTestCase {
         XCTAssertEqual(UInt8(bitPattern: result[2]), 0x83)
     }
     
+    func test_fileSystemAttributes() {
+        let fm = FileManager.default
+        let path = NSTemporaryDirectory()
+        
+        do {
+            let attrs = try fm.attributesOfFileSystem(forPath: path)
+            
+            XCTAssertTrue(attrs.count > 0)
+            
+            let systemFreeSize = attrs[.systemFreeSize] as? NSNumber
+            XCTAssertGreaterThan(systemFreeSize!.int64Value, 0)
```

Comment:
might be good to XCTAssertNotNil on the result before using ! that way any failures (in any refactoring) would immediately be visible